ks/vojtux_common.ks

```diff
@@ -179,10 +179,6 @@ rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-rpmfusion-free-fedora-*-primary
 echo "Importing RPM Fusion keys"
 rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-rpmfusion-nonfree-fedora-*-primary
 
-#installing ocrdesktop
```

Comment:
Thanks for removing those lines, but please also add the new package into the %packages section of kickstart.

Reply:
Package built in COPR and added to KS.